src/event-store/event-store.ts

```diff
@@ -35,6 +35,10 @@ export class EventStore {
     return await this.client.getAllStreamEvents(streamName);
   }
 
+  async deleteStream(streamName: string, hardDelete?: boolean) {
```

Comment:
Deze wordt nog nergens aangeroepen, wel? _(ik weet ook niet of we dat moeten willen, hoor ...)_

Reply:
Klopt, ik wist ook niet hoe we uberhaupt deze kunnen aanroepen. dat moet lijkt mij pas gebeuren nadat de mongodb bijgewerkt is. Maar hij bestaat nu ieg voor als we m ooit willen gebruiken ;)